src/v0/destinations/braze/util.js

```diff
@@ -634,6 +634,10 @@ function getPurchaseObjs(message) {
       parseInt(quantity, 10),
       timestamp,
     );
+    const extraProperties = _.omit(product, ['product_id', 'sku', 'price', 'quantity', 'currency']);
```

Comment:
if (Config.sendPurchaseEventWithExtraProperties) {
do omit logic then add to purchase object

doing omit without this config flag is useless right?
}

Reply:
I am also checking the `extraProperties` length along with config hence I am extracting it outside

Reply:
but omitting operation itself is not need right if condition is not met?